spec/bump_spec.rb

```diff
@@ -421,6 +421,15 @@ module Foo
     end
   end
 
+  context 'verify private class mothods' do
```

Comment:
Used the test to verify that private notation had an effect or not. 

Before this change the class only had 2 private class methods - inherited from Object
After change there was 19 private methods